src/integration-test/routes.ts

```diff
@@ -0,0 +1,72 @@
+import { expect } from 'chai';
+import CharlesClient from './charles-client';
+import { AccessCode, Route } from './types';
+
+const routes: Route[] = [
+  {
+    description: 'getProjects',
+    request: (me: CharlesClient, other: CharlesClient) => {
+      expect(me.teamId).to.exist;
+      expect(other.teamId).to.exist;
+      return me.getProjects(other.teamId);
+    },
+    accessMatrix: getMatrix('0', '0'),
+  },
+  {
+    description: 'getProject',
+    request: (me: CharlesClient, other: CharlesClient) => {
+      expect(me.lastProject).to.exist;
+      expect(other.lastProject).to.exist;
+      return me.getProject(other.lastProject!.id);
+    },
+    accessMatrix: getMatrix('0', '0'),
+  },
+  {
+    description: 'view deployment',
+    request: (me: CharlesClient, other: CharlesClient) => {
+      expect(me.lastDeployment).to.exist;
+      expect(other.lastDeployment).to.exist;
+      return me.fetch(other.lastDeployment!.url + '/index.html');
+    },
+    accessMatrix: [
```

Comment:
I'll add a better description to the code, but the `teamTypes` array defines both the row and column ordering, i.e
```typescript
export const teamTypes: TeamType[] = [
  'anonymous',
  'regular',
  'admin',
  'open',
];
```
So there's no "open" user, but *some* user belonging to a team of type 'open'.